src/main/java/com/bizo/awsstubs/services/dynamodb2/AmazonDynamoDBStub.java

```diff
@@ -728,6 +740,62 @@ public void addRows(Collection<Map<String, AttributeValue>> rows) {
       return matches;
     }
 
+    public List<Map<String, AttributeValue>> query(Map<String, Condition> keyConditions,
```

Comment:
Is this query overload part of the interface? If so, please add an @Override, otherwise make it private?